photon-client/package.json

```diff
@@ -1,50 +0,0 @@
-{
-  "name": "photonclient",
```

Comment:
Why was all of photon client deleted?

Reply:
I had to branch out from the new master so I copied the code from the other guys branch and the one I worked on some to the new one, its overall the same just with the cropping tab